.github/workflows/api_clients_ci.yml

```diff
@@ -0,0 +1,128 @@
+name: API Clients CI
+
+on:
+  pull_request:
+    branches: [ main ]
+    paths:
+      - 'api/**'
+      - '.github/workflows/api_clients_ci.yml'
+  workflow_dispatch:
+
+env:
+  CRATE: algokit_transact
+
+jobs:
+  setup:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      
+      - name: Setup Bun
+        uses: oven-sh/setup-bun@v2
+        with:
+          bun-version: latest
+
+      - name: Setup api tools
+        working-directory: api
+        run: bun install
+
+      - name: Generate TypeScript client
+        working-directory: api
+        run: bun run scripts/generate-algod-clients.ts --target typescript
+      
+      - name: Upload TypeScript client
+        uses: actions/upload-artifact@v4
+        with:
+          name: typescript-client
+          path: api/api_clients/typescript
+          retention-days: 1
+
+      - name: Generate Python client
+        working-directory: api
+        run: bun run scripts/generate-algod-clients.ts --target python
+      
+      - name: Upload Python client
+        uses: actions/upload-artifact@v4
+        with:
+          name: python-client
```

Comment:
Is the plan still to publish a seperate client for algod, kmd, indexer?
If so this should probably we named something like `algokit-algod-api-py` and typescript should follow the convention.